tools/protoc-plugin/src/test/java/io/spine/tools/protoc/given/CodeGeneratorRequestGiven.java

```diff
@@ -65,38 +68,46 @@ private CodeGeneratorRequestGiven() {
     }
 
     /**
-     * Creates a Base64-encoded version of a {@link SpineProtocConfig} out of the supplied
-     * {@code GeneratedMethods} and a default instance of {@code GeneratedInterfaces}.
+     * Creates a {@link SpineProtocConfig} out of the supplied {@code GeneratedMethods} and
+     * a default instance of {@code GeneratedInterfaces} and stores it in the supplied {@code path}.
      *
-     * @see #encodedProtocConfig(GeneratedInterfaces, GeneratedMethods)
+     * @return configuration file absolute path
+     * @see #protocConfig(GeneratedInterfaces, GeneratedMethods, File)
      */
-    public static String encodedProtocConfig(GeneratedMethods methods) {
-        return encodedProtocConfig(GeneratedInterfaces.withDefaults(), methods);
+    public static Path protocConfig(GeneratedMethods methods, Path configPath) {
+        return protocConfig(GeneratedInterfaces.withDefaults(), methods, configPath);
     }
 
     /**
-     * Creates a Base64-encoded version of a {@link SpineProtocConfig} out of the supplied
-     * {@code GeneratedInterfaces} and a default instance of {@code GeneratedMethods}.
+     * Creates a {@link SpineProtocConfig} out of the supplied {@code GeneratedInterfaces} and
+     * a default instance of {@code GeneratedMethods} and stores it in the supplied {@code path}.
      *
-     * @see #encodedProtocConfig(GeneratedInterfaces, GeneratedMethods)
+     * @return configuration file absolute path
+     * @see #protocConfig(GeneratedInterfaces, GeneratedMethods, File)
      */
-    public static String encodedProtocConfig(GeneratedInterfaces interfaces) {
-        return encodedProtocConfig(interfaces, GeneratedMethods.withDefaults());
+    public static Path protocConfig(GeneratedInterfaces interfaces, Path configPath) {
+        return protocConfig(interfaces, GeneratedMethods.withDefaults(), configPath);
     }
 
     /**
-     * Creates a Base64-encoded version of a {@link SpineProtocConfig} out of the supplied
-     * {@code GeneratedInterfaces} and {@code GeneratedMethods}.
+     * Creates a {@link SpineProtocConfig} out of the supplied {@code GeneratedInterfaces} and
+     * {@code GeneratedMethods} and stores it in the supplied {@code path}.
+     *
+     * @return configuration file absolute path
      */
-    public static String encodedProtocConfig(GeneratedInterfaces interfaces,
-                                             GeneratedMethods methods) {
-        SpineProtocConfig protocConfig = SpineProtocConfig
+    public static Path
+    protocConfig(GeneratedInterfaces interfaces, GeneratedMethods methods, Path configPath) {
```

Comment:
Please reduce the param count. Seems like this method does 2 different things: building and writing.